web/src/bootstrap_typeahead.js

```diff
@@ -356,7 +356,7 @@ Typeahead.prototype = {
         });
 
         $items.first().addClass("active");
-        this.$menu.html($items);
+        this.$menu.empty().append($items);
```

Comment:
Is this helpful in any way? It seems like it's just working around the linter.

Reply:
<img width="833" alt="image" src="https://github.com/zulip/zulip/assets/5634097/cfdf2d52-130c-4101-85be-748beab79e57">

Reply:
It’s actually working around the TypeScript declarations, which only [permit](https://unpkg.com/browse/@types/jquery@3.5.29/JQuery.d.ts#L5287) `.html(…)` to accept a `string` or `HTMLElement` argument (or function returning those), not `JQuery`. The behavior with a `JQuery` argument is also [undocumented](https://api.jquery.com/html/#html2).